### PR TITLE
[CSL-730] Documentation and QOL improvements

### PR DESCRIPTION
**Pull request by esezen**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+PYTHONPATH=$PYTHONPATH:.
```
Comment (esezen) [inline .env:1]: I had to check in the .env file to include this. We are not storing anything confidential so I think it's fine for now

File: Makefile [hunk 2]
```diff
@@ -19,3 +19,8 @@ install: Makefile
 	pipenv install --dev
 	pipenv run pre-commit install --hook-type pre-commit --hook-type pre-push
 
+build: Makefile
```
Comment (esezen) [inline Makefile:22]: Added these here so we don't have to memorize/document

File: setup.py [hunk 3]
```diff
@@ -15,5 +19,7 @@
     install_requires=[
         'requests~=2.26'
     ],
-    packages=find_packages(exclude=("tests",)),
+    packages = find_packages(exclude=["tests.*", "tests"]),
```
Comment (esezen) [inline setup.py:22]: Previous command was still including tests in the dist

File: setup.py [hunk 4]
```diff
@@ -15,5 +19,7 @@
...
+    long_description=long_description,
```
Comment (esezen) [inline setup.py:23]: Use README.md for pypi description

**Review by sblaurock — APPROVED** [review]
Very nice!

**Pull request merged**